SU-HSDO/suhumsci
# Release: FourKitchens Sprint 9

* shs-56: add hero overlay animation for certain heroes (#1171)

# READY FOR REVIEW

## Summary
- Limits animation styles to specific hero's (Gradient and Overlay)

## Need Review By (Date)
9/20

## Urgency
med

## Steps to Test
Codeception tests:
1. Verify this PR run's the tests and they pass.
2. run Codeception tests for permissions: blt codeception --group=permissions and verify that all tests pass.
3. run Codeception tests for content: blt codeception --group=content and verify that all tests pass.

Accordion Tests:
1. Go to the Accordions page (usually /accordions). Edit the Layout (tab at top of page). Go to the Main Region and click Configure (screenshot below). In the 'Field Formatter Class' text box, add hb-accordion-secondary. Save the layout.
2. Verify the color palette for traditional and colorful themes reflects the following changes:
    - In the sitting state, the outline is the secondary color with the standard body text color for heading/icon.
   -  Hover animates to fill the box solid with primary color background and white heading/icon.
    - On click, it returns to heading/icon in text color, but outline is now primary. Primary outline wraps around the entire content block.
    - On hover of heading/icon block in the expanded state, heading/icon switches to secondary color.

Hero changes (verified with Joey over zoom but for your purposes here):
1. Test for both colorful and traditional: 'Enable Animation Enhancements' in appearance settings
2. Go to this page: /components/hero-images-main-region and edit the page.
3. Add a Hero (overlay) component to the FULL-WIDTH REGION at the top.
4. Add a Hero Layered Slider to the main body region.
5. Save the page and verify that the text displays for the Hero (Gradient) and the Hero (Overlay) components in the body and the main full width region.
6. Verify the 'Hero Layered Slider' in the main body region does not animate. The spotlight sliders and hero no text should also not animate.
7. Edit your hero page and replace the full width region hero with the Hero Gradient slider. Verify it animates.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
